controllers/topics/inviteMembers.js

```diff
@@ -39,6 +56,28 @@ module.exports = async (req, res) => {
           return res.status(400).json(response);
         }
 
+        const invitations_msg = `${inviter.username} invited you to join ${topics.name} community`;
+
+        //topic invitation message primary chat
+        const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
+
+        const channel = await client.channel(
+          CHANNEL_TYPE_STRING.TOPIC_INVITATION,
+          `${user.username}_${topics.name}`,
+          {
+            name: `#${topics.name}`,
+            created_by_id: req?.userId,
+            members: [user_id],
+            channel_type: 5,
+            channel_image: topics.icon_path,
+            channelImage: topics.icon_path,
+            image: topics.icon_path
+          }
+        );
+        await channel.create();
+        await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
+        //end topic invitation message primary chat
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code creates a new channel for the topic invitation and sends a message to the invited user. It's important to handle any potential errors that might occur during this process. Consider wrapping this block in a try-catch statement to handle any exceptions that might be thrown when creating the channel or sending the message.
```diff
+        try {
           const invitations_msg = `${inviter.username} invited you to join ${topics.name} community`;
  
           //topic invitation message primary chat
           const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
  
           const channel = await client.channel(
             CHANNEL_TYPE_STRING.TOPIC_INVITATION,
             `${user.username}_${topics.name}`,
             {
               name: `#${topics.name}`,
               created_by_id: req?.userId,
               members: [user_id],
               channel_type: 5,
               channel_image: topics.icon_path,
               channelImage: topics.icon_path,
               image: topics.icon_path
             }
           );
           await channel.create();
           await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
           //end topic invitation message primary chat
+        } catch (error) {
+          console.error('Error creating channel or sending message:', error);
+          return res.status(500).json({ success: false, message: 'Error creating channel or sending message' });
+        }
```

89:
The change in the title of the notification to use `invitations_msg` is a good practice as it ensures consistency between the message sent in the chat and the notification.
```diff
+            title: invitations_msg,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->